src/Integration/LocalServices/IProjectSystemHelper.cs

```diff
@@ -59,5 +59,31 @@ internal interface IProjectSystemHelper : ILocalService
         /// Returns the <seealso cref="IVsHierarchy"/> for a <see cref="Project"/>
         /// </summary>
         IVsHierarchy GetIVsHierarchy(Project dteProject);
+
+        /// <summary>
+        /// Returns the currently selected projects in the active solution.
+        /// </summary>
```

Comment:
And what about `IEnumerable<Project> GetSolutionProjects` and `IEnumerable<Project> GetFilteredSolutionProjects`? They don't state that they'll never return null (and they don't ever return null).
Method that return `IEnumerable<T>` should never return null IMO, if they do then it's a bug.